exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/References.kt

```diff
@@ -236,41 +237,88 @@ private fun <ID : Comparable<ID>> List<Entity<ID>>.preloadRelations(
         }
     }
 
+    fun Entity<*>.getReferenceId(
+        delegateRefColumn: Column<*>,
+        refColumns: Map<Column<*>, Column<*>>,
+        isNotCompositeIdReference: Boolean
```

Comment:
Also minor, but negatives in arguments a bit harder to read, usually better to name without them (`isCompositeIdReference` in this case)